catalog-info.yaml

```diff
@@ -36,8 +36,7 @@ spec:
       provider_settings:
         build_pull_requests: true
         build_branches: true
-      cancel_intermediate_builds: true
-      cancel_intermediate_builds_branch_filter: '!main'
+      cancel_intermediate_builds: false
```

Comment:
We don't want to cancel intermediate builds to make sure we always tear down all serverless projects.